content/en/blog/2024/docs-localized.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 中文, 日本語, Português — OpenTelemetry website goes multilingual
```

Comment:
I think we should go with localized blog posts if possible and avoid the multilingual title, which reads kind of Babel-like. :)

Reply:
```suggestion
title: OpenTelemetry website goes multilingual!
```